main/domoticz.cpp

```diff
@@ -353,7 +353,7 @@ void daemonize(const char *rundir, const char *pidfile)
 	{
 		return GetModuleFileNameA(NULL, pathName, (DWORD)pathNameCapacity);
 	}
-#elif defined(__linux__) || defined(__CYGWIN32__) /* elif of: #if defined(_WIN32) */
+#elif defined(__linux__) || defined(__CYGWIN32__) || defined(__NetBSD__) /* elif of: #if defined(_WIN32) */
```

Comment:
Do you need to run this on NetBSD<8.0? New code should rather use sysctl(3) similar to FreeBSD below.

Reply:
Ah, yes - I see we're almost compatible with FreeBSD for this lookup. Fixing now.